package.json

```diff
@@ -87,6 +87,7 @@
     "@types/got": "^9.6.12",
     "@types/inquirer": "^9.0.7",
     "@types/ip": "^1.1.3",
+    "@types/js-yaml": "^4.0.9",
```

Comment:
If we can use `yaml` instead of `js-yaml` that would be better.  Nathan wants us to switch to using `yaml` because it is more maintained.   `js-yaml` hasn't been updated in about 4 years.

Reply:
Isn't js-yaml more popular? 
Also if it's stable there shouldn't really be any updates since the core functionality doesn't change I wouldn't call it stale

Reply:
also js-yaml seems more stable